### PR TITLE
Refactor runner's code

### PR DESCRIPTION
**Pull request by stgleb**
Remove args from command
Change command name to s cript
Change Runner interface

### CODE REVIEW

**Comment by stgleb** [comment]
@saheienko Sure, I will

**Comment by stgleb** [comment]
@saheienko  I've looked once again. How do you want test looks like? Because SshRunner tightly coupled to ssh.Client there is no way to avoid direct calls to ssh. Do you want me to decouple client from SshRunner by interface?

**Comment by saheienko** [comment]
```
+func (r *Runner) Run(c *command.Command) (err error) {
 	if r.client == nil {
 	if r.client == nil {
 		return errors.New("not connected")
 		return errors.New("not connected")
 	}
 	}
 
 
-	cmd := strings.TrimSpace(c.FullCommand())
+	cmd := strings.TrimSpace(c.Script)
 	if cmd == "" {
 	if cmd == "" {
 		return nil
 		return nil
 	}
 	}
```
Do you need a ssh.Client for tesging this block of code?

File: pkg/runner/command/command.go [hunk 1]
```diff
@@ -3,31 +3,23 @@ package command
...
-	Command string
-	Args    []string
+	Script string
 
 	Out io.Writer
 	Err io.Writer
 }
 
-func NewCommand(ctx context.Context, cmd string, args []string, out, err io.Writer) *Command {
+func NewCommand(ctx context.Context, script string, out, err io.Writer) *Command {
```
Comment (saheienko) [inline pkg/runner/command/command.go]: How do you plan to run this script?

Comment (stgleb) [reply pkg/runner/command/command.go]: @yegorch could you elaborate on that?

Comment (stgleb) [reply pkg/runner/command/command.go]: @saheienko We  just pass valid script where each command goes line by line and it is executed entirely.

**Pull request merged**